### PR TITLE
updates to steps and procedures

### PR DESCRIPTION
**Pull request by catlook**
added and refined steps so that contributors know how to contribute.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -56,112 +56,132 @@ with the master How-To repository. If the status says `This branch is X commits
...
+   `git checkout -b my-new-server-article`
+
+3. Create a new directory within ```/content/how-to/**product**``` that contains
+   the initial ```.index.md``` by using the following command:
+
+   `hugo new --kind support-bundle how-to/**product**/article-title`
+
+   For example, the command, `hugo new --kind support-bundle how-to/cloud-servers/create-a-new-cloud-server`,
+   creates a new directory, **create-a-new-cloud-server**, in the **cloud-server** product directory. The new 
+   direcory contains four files:
```
Comment (robb-romans) [inline CONTRIBUTING.md:97]: typo

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -56,112 +56,132 @@ with the master How-To repository. If the status says `This branch is X commits
...
+    a. Add your new content and image files, if any:
+    
+       `git add index.md`
+       `git add Picture1.png`
+       
+    b. Commit the file:
+    
+       `git commit -m "Created new cloud server article"`
+       
+    c. Push the chnage
```
Comment (robb-romans) [inline CONTRIBUTING.md:137]: typo

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -56,112 +56,132 @@ with the master How-To repository. If the status says `This branch is X commits
...
+       
+    e. In the How-to repo, click **Pull Requests**.
+    
+    f. At the top, you should see a message with your branch name and a green **Compare & pull request** button. Click it.
+    
+    g. Click **Compare across forks**.
+    
+    h. `base-repository` should default to `rackerlabs/support-how-to` and `base` should default to `master`. Click the
+       down arrow on `head-repository` and select your clone of the support-how-to repo. Click the down arrow on
+       `compare` and select your branch. Click **Create new pull request**. Add any comments decsibing your change.
```
Comment (robb-romans) [inline CONTRIBUTING.md:152]: typo

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -56,112 +56,132 @@ with the master How-To repository. If the status says `This branch is X commits
...
+       Finally, click **Create pull request**.
+
+7. After the build completes, preview it by clicking the **Details** link next
+   to *Deploy preview ready!* in the **All Checks Passed** section of the GitHub PR.
+   This action takes you to the staging location. Click **How-To** and then paste the
+   permalink for your article (from the **index.md** metedata) at the end of the URL
```
Comment (robb-romans) [inline CONTRIBUTING.md:158]: typo

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -196,25 +211,28 @@ the article folder.
...
 
-8. Click **Create pull request**.
+9. Click **Create pull request**.
 
-9. After the build completes, preview it by clicking the **Details** link next
-to *Deploy preview ready!* in the **All Checks Passed** section of the GitHub PR.
+10. After the build completes, preview it by clicking the **Details** link next
+   to *Deploy preview ready!* in the **All Checks Passed** section of the GitHub PR.
+   This action takes you to the staging location. Click **How-To** and then paste the
+   permalink for your article (from the **index.md** metedata) at the end of the URL
```
Comment (robb-romans) [inline CONTRIBUTING.md:219]: typo

**Review by robb-romans — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (catlook) [reply CONTRIBUTING.md:97]: got it!

File: CONTRIBUTING.md (hunk 2 above)
Comment (catlook) [reply CONTRIBUTING.md:137]: yes.

File: CONTRIBUTING.md (hunk 3 above)
Comment (catlook) [reply CONTRIBUTING.md:152]: doh

File: CONTRIBUTING.md (hunk 4 above)
Comment (catlook) [reply CONTRIBUTING.md:158]: metedata?   Maybe I was hungry.

File: CONTRIBUTING.md (hunk 5 above)
Comment (catlook) [reply CONTRIBUTING.md:219]: copy garbage, get garbage!

**Comment by robb-romans** [comment]
Please add a more descriptive commit message

**Review by stephamon — APPROVED** [review]

**Comment by catlook** [comment]
I updated my original description to "added and refined steps so that contributors know how to contribute."  Is that sufficient?

**Review by robb-romans — APPROVED** [review]

**Comment by robb-romans** [comment]
thanks

**Pull request merged**